### ISSUE TITLE
Processos em background continuam quando a aplicação é fechada ou está pausada

### ISSUE THREAD

**Issue by fmsouza** [RioBus/android#39]

**Comment by georgerappel** [comment]
Ao buscar linhas inexistentes (ex: digitando só letras), o balão de erro também fica reaparecendo.

**Comment by georgerappel** [comment]
Resolvido pelo commit aa2e61d3bea1c9a785e20f95461e1ef9255a0809?

Mas ainda é preciso saber se ele continua buscando os dados em background. Se deixar o aplicativo aberto pelo resto do dia vai consumir muita bateria fazendo download das linhas a cada X segundos.

@limazix

**Comment by limazix** [comment]
Sim, ainda ficam. Vamos corrigir isso com a finalização do Composite. Por hora, se der algum erro na requisição ele não faz a seguinte e mostra uma mensagem na Snackbar.
